src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstallerProjectEvents.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using NuGet.ProjectManagement;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Contains batch events to be raised when performing multiple packages install/ uninstall in a project
+    /// </summary>
+    [Export(typeof(IVsPackageInstallerProjectEvents))]
+    [Export(typeof(VsPackageInstallerProjectEvents))]
```

Comment:
You can remove this export, only the interface is needed. It doesn't hurt anything to have this, but there's no real way for a 3rd party to import this. The interface on the other hand is an embedded interop type and will not change between assembly versions.